### ISSUE TITLE
How to train Faster R-CNN on my own custom dataset?

### ISSUE THREAD

**Issue by meshghi** [facebookresearch/Detectron#1028]
Hello. I want to:

1- train Faster RCNN on my own custom dataset.
2- use the pre-trained Faster RCNN on the VOC2007 as the initial weights to train it then on my own custom dataset.
3- modify the RPN network with different loss functions that I have come up with.

I will greatly appreciate it if you could help me/let me know how to achieve the above goals.

**Comment by under-score** [comment]
Have a look at [towardsdatascience.com](https://towardsdatascience.com/how-to-train-detectron2-on-custom-object-detection-data-be9d1c233e4) for custom object detection.

**Comment by s4nyam** [comment]
Still Looking for some productive references, not literature, but proper guidelines to code. Thanks :)

**Comment by aymuos15** [comment]
https://debuggercafe.com/how-to-train-faster-rcnn-resnet50-fpn-v2-on-custom-dataset/

This link seemed fairly helpful (for question 1). At least provides some form of intuition as to how structure your pipeline.
